bin/nav

```diff
@@ -70,9 +70,41 @@ def make_argparser():
         subp.add_argument("service", nargs="*", default=all_services)
         subp.set_defaults(func=func)
 
+    _add_bespoke_subparsers(subparsers)
```

Comment:
I'm not too fond of invisible side effects. I would have returned `subparsers` for explicitness.

Reply:
I would tend to agree, but since `argparse` produces the subparsers object for us to manipulate afterwards, I'm not sure how I would refactor this.